code/game/machinery/doors/poddoor/shutters/shutters.dm

```diff
@@ -152,7 +152,7 @@
 		return
 
 /obj/structure/machinery/door/poddoor/shutters/almayer/uniform_vendors/attackby(obj/item/attacking_item, mob/user)
-	if(HAS_TRAIT(attacking_item, TRAIT_TOOL_CROWBAR))
+	if(HAS_TRAIT(attacking_item, TRAIT_TOOL_CROWBAR) || attacking_item.pry_capable)
```

Comment:
From what I can see only jack forgets to change that var when flipping to wrench, ideally you would want to fix that but this is fine too